homeassistant/components/sensor/sigfox.py

```diff
@@ -0,0 +1,140 @@
+"""
+Sensor for SigFox devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.sigfox/
+"""
+import logging
+import datetime
+import json
+import voluptuous as vol
+import requests
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = datetime.timedelta(seconds=30)
+API_URL = 'https://backend.sigfox.com/api/'
+CONF_API_LOGIN = 'api_login'
+CONF_API_PASSWORD = 'api_password'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_API_LOGIN): cv.string,
+    vol.Required(CONF_API_PASSWORD): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the sigfox sensor."""
+    api_login = config[CONF_API_LOGIN]
+    api_password = config[CONF_API_PASSWORD]
+    sigfox = SigfoxAPI(api_login, api_password)
+    auth = sigfox.auth
+    devices = sigfox.devices
+
+    sensors = []
+    for device in devices:
+        sensors.append(SigfoxDevice(device, auth))
+    add_devices(sensors, True)
+
+
+def epoch_to_datetime(epoch_time):
```

Comment:
Is there nothing already available in `util.dt`?

Reply:
[I don't see anything unfortunately](https://github.com/home-assistant/home-assistant/blob/fdf93d18298c89714e438dc5dfbba858f29de18e/homeassistant/util/dt.py). `.isoformat()` is how I was advised to format the time in the `folder_watcher` recently